src/main/java/codoc/ui/PersonListPanel.java

```diff
@@ -31,6 +31,10 @@ public PersonListPanel(MainWindow mainWindow) {
         personListView.setCellFactory(listView -> new PersonListViewCell());
     }
 
+    public void showLastItem() {
+        personListView.scrollTo(personListView.getItems().size() - 1);
```

Comment:
Add Command puts the person to the last entry.
Since ModelManager's addPerson clears all existing predicates to show all person, this would definitely scroll to the last person.